docs/v2_rewrite_implementation_plan/README.md

```
@@ -0,0 +1,8 @@
+# V2 Rewrite Implementation Plan Docs
+
+This folder contains planning documents for the V2 rewrite and Phase 3 E2E coverage.  
+See [phase-3-e2e-test-coverage-plan.md](phase-3-e2e-test-coverage-plan.md) for a living checklist of E2E scenarios.
+
+Related:  
+- Agent API v2 specs: [.kiro/specs/phase-2b-agent-api-v2/](.kiro/specs/phase-2b-agent-api-v2/)  
+- CI setup: [.github/workflows/copilot-setup-steps.yml](.github/workflows/copilot-setup-steps.yml), [.github/workflows/ci.yml](.github/workflows/ci.yml) (pnpm pinned)
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)